### PR TITLE
add rabbitmq  testcase

### PR DESCRIPTION
**Pull request by withlin**
as title

### CODE REVIEW

**Comment by ascrutae** [comment]
The jenkins job show the rabbitmq-scenario is not testcase project, and it miss `testcase.yml` file. and you can following other scenario project to resolve this issue.

**Comment by withlin** [comment]
Thank you for your reminder. i am resloved it.

**Comment by ascrutae** [comment]
Is the `3-management:latest` image visible?

**Comment by withlin** [comment]
no, i am fixed it.

File: rabbitmq-scenario/config/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+version: '2.1'
+services:
+  rabbitmq-server:
+    image: rabbitmq:latest
+    expose:
+    - "5672"
+    ports:
+      - "5672:5672"
```
Comment (ascrutae) [inline rabbitmq-scenario/config/docker-compose.yml]: Remove the mapping ports. Otherwise it will cause the exception that the port is already allocated.

**Comment by ascrutae** [comment]
@WithLin  Could you test it in your local environment and fixed it？Here is the error log:
```
rabbitmq-scenario_1  | java.lang.NullPointerException: null
rabbitmq-scenario_1  | 	at java.util.Properties$LineReader.readLine(Properties.java:434) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at java.util.Properties.load0(Properties.java:353) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at java.util.Properties.load(Properties.java:341) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at com.rabbitmq.client.impl.ClientVersion.getVersionFromPropertyFile(ClientVersion.java:53) ~[amqp-client-5.1.0.jar!/:5.1.0]
rabbitmq-scenario_1  | 	at com.rabbitmq.client.impl.ClientVersion.<clinit>(ClientVersion.java:36) ~[amqp-client-5.1.0.jar!/:5.1.0]
rabbitmq-scenario_1  | 	at com.rabbitmq.client.impl.AMQConnection.defaultClientProperties(AMQConnection.java:75) [amqp-client-5.1.0.jar!/:5.1.0]
rabbitmq-scenario_1  | 	at com.rabbitmq.client.ConnectionFactory.<init>(ConnectionFactory.java:103) [amqp-client-5.1.0.jar!/:5.1.0]
rabbitmq-scenario_1  | 	at test.apache.skywalking.testcase.rabbitmq.controller.CaseController.rabbitmqCase$original$N89Yt4QD(CaseController.java:63) [classes!/:na]
rabbitmq-scenario_1  | 	at test.apache.skywalking.testcase.rabbitmq.controller.CaseController.rabbitmqCase$original$N89Yt4QD$accessor$rc7qH3xM(CaseController.java) [classes!/:na]
rabbitmq-scenario_1  | 	at test.apache.skywalking.testcase.rabbitmq.controller.CaseController$auxiliary$uzmiS511.call(Unknown Source) [classes!/:na]
rabbitmq-scenario_1  | 	at org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstMethodsInter.intercept(InstMethodsInter.java:93) [skywalking-agent.jar:5.0.0-RC-SNAPSHOT]
rabbitmq-scenario_1  | 	at test.apache.skywalking.testcase.rabbitmq.controller.CaseController.rabbitmqCase(CaseController.java) [classes!/:na]
rabbitmq-scenario_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_181]
rabbitmq-scenario_1  | 	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205) [spring-web-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest$original$HrUe5JmC(InvocableHandlerMethod.java:133) [spring-web-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest$original$HrUe5JmC$accessor$hjQh88sv(InvocableHandlerMethod.java) [spring-web-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.method.support.InvocableHandlerMethod$auxiliary$WjWmIJfZ.call(Unknown Source) [spring-web-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstMethodsInter.intercept(InstMethodsInter.java:93) [skywalking-agent.jar:5.0.0-RC-SNAPSHOT]
rabbitmq-scenario_1  | 	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java) [spring-web-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:116) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:827) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:738) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:85) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:963) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:897) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:970) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:861) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:622) [tomcat-embed-core-8.5.11.jar!/:8.5.11]
rabbitmq-scenario_1  | 	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:846) [spring-webmvc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
rabbitmq-scenario_1  | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729) [tomcat-embed-core-8.5.11.jar!/:8.5.11]

```

**Comment by withlin** [comment]
hi,Is this the error of the api address url  report requesting the test case? Still something else?

**Pull request closed without merging**